evm/client/chain_client.go

```diff
@@ -374,20 +373,16 @@ func (c *chainClient) PendingNonceAt(ctx context.Context, account common.Address
 }
 
 func (c *chainClient) SendTransaction(ctx context.Context, tx *types.Transaction) error {
-	var result *SendTxResult
 	if c.chainType == chaintype.ChainHedera {
 		activeRPC, err := c.multiNode.SelectRPC()
 		if err != nil {
 			return err
 		}
-		result = activeRPC.SendTransaction(ctx, tx)
-	} else {
-		result = c.txSender.SendTransaction(ctx, tx)
```

Comment:
@jmank88 the separation between Hedera and other chains was intentional. On hedera we only want to broadcast transaction to single RPC, while on other chains we want to broadcast to all available.

Reply:
The special case Hedera logic is still there